### ISSUE TITLE
Associate Combobox error message with input

### ISSUE THREAD

**Issue by thuey** [dequelabs/cauldron#1296]
Currently, the `Combobox`'s error message does not get semantically associated with the input element. It should be associated via `aria-describedby` for assistive technology

**Comment by scurker** [comment]
fixed by #1297
